receiver/purefareceiver/go.mod

```diff
@@ -3,15 +3,15 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/purefa
 go 1.18
 
 require (
-	github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.69.0
-	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.69.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/extension/bearertokenauthextension v0.70.0
```

Comment:
This change is still bugging me, as the current main is already at v0.70.0. Can you please rebase this PR? I'm not expecting any changes to the go.mod/sum as part of this PR.